City-of-Pori/pori-kada
# PORIJATE-8 waste search form

To test: 
- run ./build.sh update
- run drush fra -y && drush cc all
- goto https://local.pori.fi/admin/config/system/ip_geoloc and add the Google API key
- goto https://local.pori.fi/admin/content?title=&author=&type=place&status=All&language=All&language_1=All and edit few places by
   - adding the "This place should be visible on the waste search" checkbox
   - clicking the "Etsi Visiting address perusteella" 
   - save
- goto https://local.pori.fi/admin/structure/taxonomy/waste_type/add and add few different waste types
   - remember to add reference to places
- goto https://local.pori.fi/admin/structure/taxonomy/waste/add and add few different wastes
   - remember to add reference to waste types
- Finally go to https://local.pori.fi/waste/search and see if the search works like it should
